tests/DependencyInjection/DoctrineMongoDBExtensionTest.php

```diff
@@ -369,4 +372,70 @@ public function testControllerResolver(): void
         $container->compile();
         $this->assertEquals(new MapDocument(null, null, null, [], null, null, null, true), $container->get('controller_resolver_defaults'));
     }
+
+    public function testTransactionalFlushConfigurationWhenNotSupported(): void
+    {
+        if (method_exists(ODMConfiguration::class, 'setUseTransactionalFlush')) {
```

Comment:
This is a weak test. If the method is renamed, the test is skipped while it should fail.

I think it would be more robust to use `Composer\InstalledVersions`:

```php
Composer\InstalledVersions::satisfies(new VersionParser, 'doctrine/mongodb-odm', '>=2.7@dev')
```

Reply:
Updated. I wasn't even aware this existed. This is simply lovely :)

Reply:
I see [3 skipped tests](https://github.com/doctrine/DoctrineMongoDBBundle/actions/runs/7584953100/job/20659752583?pr=814#step:9:15), is the version contraint correct?